### PR TITLE
New listing view for users, ontologies and projects

### LINKED ISSUE #99: ontologies
A project can have more than one ontology (when issue #97 is done). So we need an additional tab or a replacement for the "resources" tab. And we need a button to add more ontologies.

The webapi isn't ready yet to create ontologies in a direct way, but we need a solution right now. As @lrosenth suggested, we have to work with turtle (ttl) files. It means, that we need a translation layer to get json from turtle or to create turtle from json. The solution could be N3 which can handle this.

Todo:
- [x] New tab for ontologies incl. add button

### PR DESCRIPTION
**Pull request by kilchenmann**
resolves #99

### CODE REVIEW

File: src/app/view/dashboard/project/project-team/project-team.component.spec.ts [hunk 1]
```diff
@@ -79,12 +79,98 @@ describe('ProjectTeamComponent', () => {
...
+                    'description': 'Bibliography',
+                    'institution': null,
+                    'logo': null,
+                    'dataNamedGraph': 'http://www.knora.org/data/biblio',
+                    'id': 'http://data.knora.org/projects/DczxPs-sR6aZN91qV92ZmQ',
+                    'status': true,
+                    'selfjoin': false,
+                    'keywords': null,
+                    'longname': 'Bibliography',
+                    'ontologyNamedGraph': 'http://www.knora.org/ontology/biblio'
```
Comment (subotic) [inline src/app/view/dashboard/project/project-team/project-team.component.spec.ts]: You will need to change the test data: `ontologyNamedGraph` is now `ontologies`

File: src/app/view/modules/framework/framework-for-listings/framework-for-listings.component.ts [hunk 2]
```diff
@@ -103,9 +129,9 @@ export class FrameworkForListingsComponent implements OnInit, OnChanges {
 
     ngOnInit() {
         // bad hack to get the project admin information
-        this.loggedInAdmin = (sessionStorage.getItem('admin') !== null);
-        // end of bad hack. TODO: we have to find a better solution
-
+        if (localStorage.getItem('currentUser') !== null) {
```
Comment (subotic) [inline src/app/view/modules/framework/framework-for-listings/framework-for-listings.component.ts]: Are we using localStorage or sessionStorage?

Comment (kilchenmann) [reply src/app/view/modules/framework/framework-for-listings/framework-for-listings.component.ts]: We're using sessionStorage for temporary information e.g. in project admin we store current project, current project members etc.
The localStorage is used to store the information about the logged in user called currentUser.

File: src/app/view/dashboard/project/project-team/project-team.component.spec.ts (hunk 1 above)
Comment (kilchenmann) [reply src/app/view/dashboard/project/project-team/project-team.component.spec.ts]: I will fix it

**Pull request merged**